src/Maestro/Maestro.Web/Api/v2019_01_16/Controllers/BuildsController.cs

```diff
@@ -84,6 +84,9 @@ public override async Task<IActionResult> GetBuild(int id)
                 return NotFound();
             }
 
+            List<Data.Models.BuildDependency> dependentBuilds = _context.BuildDependencies.Where(b => b.BuildId == id).ToList();
```

Comment:
Can this be moved into the query above? https://github.com/dotnet/arcade-services/pull/711/files#diff-0e88431d142c708ef3190586ba2e9d5aR79

Reply:
I originally tried to do that, but I think because the DependentBuildIds list is marked as [NotMapped], the data doesn't exist, so I need to go directly to the BuildDependencies table. When I tried to add `.Include(b => b.DependentBuildIds)` I get:

```
 cache-control: no-cache 
 date: Thu, 24 Oct 2019 15:33:12 GMT 
 expires: -1 
 pragma: no-cache 
 server: Microsoft-HTTPAPI/2.0 
 status: 500 
```

Where as with what I have here, I get response 200 with the dependencies in the response body:

```
"dependencies": [
    {
      "buildId": 388,
      "isProduct": true,
      "timeToInclusionInMinutes": 0.015093556666666667
    },
    {
      "buildId": 389,
      "isProduct": true,
      "timeToInclusionInMinutes": 0.00692308
    }
  ]
```